README.md

```diff
@@ -30,6 +30,44 @@ videojs('player_id').ready(function () {
 </script>
 ```
 
+## Known Issues
+
+1. Fairplay content cannot be played if `TextTrack`s exist. Beyond embedding them within the manifest, the only way to side load them is after the keys are updated.
+
+  ```js
+  videojs('player_id').ready(function () {
+    var player = this;
+
+    player.on('loadeddata', function () {
+      player.addRemoteTextTrack(...);
+    });
+  });
+  ```
+
+1. New Fairplay content cannot be played once MediaKeys have been attached to the video element (Unless its the same source). To get around this issue we need to clear those keys.
+
+  ```js
+  player.reset();
+
+  player.src(...);
+  ```
+
+1. Combining the issues of #1 and #2. Before you can play *another* source, you must clear out the existing `TextTrack`s.
+
+  ```js
+  player.remoteTextTracks().tracks_.forEach(function (track) {
+    player.removeRemoteTextTrack(track);
+  });
+
+  // the current `TextTrack` implementation operates on events and because of this we have to wait a
+  // tick before executing a reset, otherwise the `TextTrack`s will get picked up in its associated `Tech`.
+  setTimeout(function () {
+    player.reset();
```

Comment:
it actually doesn't properly clean up and i suspect its cached.

https://github.com/videojs/video.js/blob/master/src/js/tech/html5.js#L65?

i still believe it takes a tick for the text tracks to be removed because of the event implementation.